src/ui/image/Image.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useRef } from "react"
+
+import { Loader, Button } from "ui"
+
+import csx from "./Image.scss"
+
+namespace Image {
+    export interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
+        className?: string;
+        lowQuality: string;
+        fallback?: string;
+    }
+}
+
+const Image = ({ lowQuality, fallback, src, className, width, height, ...props }: Image.Props) => {
+    const [url, setUrl] = useState(lowQuality || src)
```

Comment:
This code doing too many things. All what you need here is just error state variable and errorHandler fn.

URL value is calculated on every render - use memo here with error as deps array and then you passing this to img src attribute.